h2/src/test/org/h2/test/TestAll.java

```diff
@@ -167,68 +167,7 @@
 import org.h2.test.synth.TestTimer;
 import org.h2.test.synth.sql.TestSynth;
 import org.h2.test.synth.thread.TestMulti;
-import org.h2.test.unit.TestAnsCompression;
-import org.h2.test.unit.TestAutoReconnect;
-import org.h2.test.unit.TestBinaryArithmeticStream;
-import org.h2.test.unit.TestBitStream;
-import org.h2.test.unit.TestBnf;
-import org.h2.test.unit.TestCache;
-import org.h2.test.unit.TestCharsetCollator;
-import org.h2.test.unit.TestClearReferences;
-import org.h2.test.unit.TestCollation;
-import org.h2.test.unit.TestCompress;
-import org.h2.test.unit.TestConnectionInfo;
-import org.h2.test.unit.TestDataPage;
-import org.h2.test.unit.TestDate;
-import org.h2.test.unit.TestDateIso8601;
-import org.h2.test.unit.TestDateTimeUtils;
-import org.h2.test.unit.TestDbException;
-import org.h2.test.unit.TestExit;
-import org.h2.test.unit.TestFile;
-import org.h2.test.unit.TestFileLock;
-import org.h2.test.unit.TestFileLockProcess;
-import org.h2.test.unit.TestFileLockSerialized;
-import org.h2.test.unit.TestFileSystem;
-import org.h2.test.unit.TestFtp;
-import org.h2.test.unit.TestGeometryUtils;
-import org.h2.test.unit.TestIntArray;
-import org.h2.test.unit.TestIntIntHashMap;
-import org.h2.test.unit.TestIntPerfectHash;
-import org.h2.test.unit.TestInterval;
-import org.h2.test.unit.TestJmx;
-import org.h2.test.unit.TestLocalResultFactory;
-import org.h2.test.unit.TestLocale;
-import org.h2.test.unit.TestMathUtils;
-import org.h2.test.unit.TestMemoryUnmapper;
-import org.h2.test.unit.TestMode;
-import org.h2.test.unit.TestModifyOnWrite;
-import org.h2.test.unit.TestNetUtils;
-import org.h2.test.unit.TestObjectDeserialization;
-import org.h2.test.unit.TestOldVersion;
-import org.h2.test.unit.TestOverflow;
-import org.h2.test.unit.TestPageStore;
-import org.h2.test.unit.TestPageStoreCoverage;
-import org.h2.test.unit.TestPattern;
-import org.h2.test.unit.TestPerfectHash;
-import org.h2.test.unit.TestPgServer;
-import org.h2.test.unit.TestReader;
-import org.h2.test.unit.TestRecovery;
-import org.h2.test.unit.TestReopen;
-import org.h2.test.unit.TestSampleApps;
-import org.h2.test.unit.TestScriptReader;
-import org.h2.test.unit.TestSecurity;
-import org.h2.test.unit.TestShell;
-import org.h2.test.unit.TestSort;
-import org.h2.test.unit.TestStreams;
-import org.h2.test.unit.TestStringCache;
-import org.h2.test.unit.TestStringUtils;
-import org.h2.test.unit.TestTimeStampWithTimeZone;
-import org.h2.test.unit.TestTools;
-import org.h2.test.unit.TestTraceSystem;
-import org.h2.test.unit.TestUtils;
-import org.h2.test.unit.TestValue;
-import org.h2.test.unit.TestValueHashMap;
-import org.h2.test.unit.TestValueMemory;
```

Comment:
Please, avoid such changes. `.*` imports are not used in H2.

Reply:
Thanks, fixed.